dotnet/src/Extensions/Planning.SequentialPlanner/FunctionViewExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Linq;
+using Microsoft.SemanticKernel.SkillDefinition;
+
+namespace Microsoft.SemanticKernel.Planning.Sequential;
+
+internal static class FunctionViewExtensions
+{
+    /// <summary>
+    /// Create a manual-friendly string for a function.
+    /// </summary>
+    /// <param name="function">The function to create a manual-friendly string for.</param>
+    /// <returns>A manual-friendly string for a function.</returns>
+    internal static string ToManualString(this FunctionView function)
+    {
+        var inputs = string.Join("\n", function.Parameters.Select(p => $"    - {p.Name}: {p.Description}"));
+        return $"  {function.ToFullyQualifiedName()}:\n    description: {function.Description}\n    inputs:\n{inputs}";
```

Comment:
Some merge changes were missed here -- I'd expect to see the old file deleted or marked a rename. I can resolve in follow up, too.

Reply:
I re-did the "git mv", the file should have the right content